ReverseString.java

```diff
@@ -0,0 +1,57 @@
+import java.util.Scanner;
+
+public class ReverseString {
+    public static void main(String[] args) {
+
+        /************************************************
+        Write a Java program to reverse a string and 
+        Write a Java program to count the letters, spaces, 
+        numbers and other characters of an input string.
+        *************************************************/
+        
+        Scanner scan = new Scanner(System.in);
+        int escape = 1;
+        int letters = 0;
+        int numbers = 0;
+        int spaces = 0;
+        int othersChar = 0;
+        while(escape == 1) {
+            System.out.println("Input a sentence: ");
+            char [] chars = scan.nextLine().toCharArray();
+
+            for(int index = chars.length-1; index >=0; index--) {
+                System.out.print(chars[index]);
+            }
+
+            System.out.print("\n"); 
+            
+            for(int index = chars.length-1; index >=0; index--) {
+                if((chars[index] > 65 && chars[index] < 90 ) || (chars[index] > 97 && chars[index] < 122)) {
```

Comment:
you want to include A and Z so chars[index] >= 65 or maybe better chars[index] >= 'A'